conan/cli/formatters/graph/graph.py

```diff
@@ -28,15 +27,7 @@ def format_graph_html(result):
     if os.path.isfile(user_template):
         with open(user_template, 'r', encoding="utf-8", newline="") as handle:
             template = handle.read()
-    error = {
-        "type": "unknown",
-        "context": graph.error,
-        "should_highlight_node": lambda node: False
-    }
-    if isinstance(graph.error, GraphConflictError):
-        error["type"] = "conflict"
-        error["should_highlight_node"] = lambda node: node.id == graph.error.node.id
```

Comment:
This was mostly dead, long time that the html template do not use this input, but the ``graph["error"]`` field instead.